AnnoyancesFilter/Other/sections/annoyances.txt

```diff
@@ -8,6 +8,7 @@
 !######### Annoying elements (not ads)
 ! SECTION: Other annoyances - Uncategorized
 !
+jam-news.net##section#donationPopup.donation-popup-box.active
```

Comment:
Only one class or ID is enough here.
```suggestion
jam-news.net##.donation-popup-box
```

Please move to AnnoyancesFilter\Widgets\sections\widgets.txt in `Donation widgets` section

Reply:
done